Tensile/Components/MAC_F32.py

```diff
@@ -47,6 +48,10 @@ def __call__(self, writer, m, innerUnroll):
             else:
                 raise RuntimeError("FMA instruction specified but not supported on {}".format(kernel["ISA"]))
 
+        dualMacEnable = 0
+        if writer.asmCaps["v_dual_fmac_f32"] and kernel["WavefrontSize"] == 32:
```

Comment:
Why WavefrontSize needs to be 32 to generate v_dual_fmac_f32?

Reply:
According to the spec, this instruction format is only legal for wave32.

Reply:
> According to the spec, this instruction format is only legal for wave32.

OK. 
I probably need to ask this in a different way.
If it is not legal, I guess asmCap will be false.
I would like to know why we need to check an extra condition besides asmCap.
I feel just checking asmCap should be enough.

Reply:
We set asmCap without checking wave32/64. The gfx11 has both wave32 and wave64 mode. Generally, we will use wave32 mode in gfx11. asmCap will be always true on gfx11 platform.